src/lib/transaction_database/transaction_database.ml

```diff
@@ -641,3 +641,52 @@ let%test_module "Transaction_database" =
 
 module T = Make (User_command) (Block_time.Time.Stable.V1)
 include T
+
+module For_tests = struct
+  open Quickcheck.Generator
+  open Let_syntax
+
+  let of_year years = Int64.of_int (years * 365 * 24 * 60 * 60 * 1000)
+
+  let populate_database ~directory num_users num_commands =
+    let keypairs = List.init num_users ~f:(fun _ -> Keypair.create ()) in
```

Comment:
instead of making keypairs here, can you make (some of) them using the `Wallets` module so they'll appear in wallet queries properly in graphql.

Maybe we can make that a flag as well, how many of the users to make using the wallets module (aka the frontend will think we own the wallets) vs. how many to make in this manner (that will simulate other people).